root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -554,6 +554,7 @@ $.widget("mb.entitylookup", $.ui.autocomplete, {
         if (arguments.length) {
             recentEntities[entityType] = _.take(arguments[0], MAX_RECENT_ENTITIES);
             localStorage("recentAutocompleteEntities", JSON.stringify(recentEntities));
+            return undefined;
```

Comment:
This acts as either a setter or a getter depending on how many parameters it is getting. It's nicer to think of it really as two functions with the same name. This means that there was never an inconsistency, but I think explicitly returning `undefined` is the best way to show this. (Comment **A**)